minerl/herobraine/hero/handlers/agent/start.py

```diff
@@ -60,36 +69,44 @@ def __init__(self, inventory: Dict[int, Dict[str, Union[str, int]]]):
 
 
 class SimpleInventoryAgentStart(InventoryAgentStart):
-    """ An inventory agentstart specification which
-    just fills the inventory of the agent sequentially.
+    """ Sets the start inventory of the agent sequentially.
+
+    Example usage:
+
+    .. code-block:: python
+
+        SimpleInventoryAgentStart([
+            {'type':'dirt', 'quantity':10},
+            {'type':'planks', 'quantity':5},
+            {'type':'log', 'quantity':1},
+            {'type':'iron_ore', 'quantity':4}
+        ])
+
     """
     def __init__(self, inventory : List[Dict[str, Union[str, int]]]):
         """ Creates a simple inventory agent start.
 
-        For example:
-
-            sias =  SimpleInventoryAgentStart(
-                [
-                    {'type':'dirt', 'quantity':10},
-                    {'type':'planks', 'quantity':5},
-                    {'type':'log', 'quantity':1},
-                    {'type':'iron_ore', 'quantity':4}
-                ]
-            )
         """
         super().__init__({
             i: item for i, item in enumerate(inventory)
         })
 
 
 class RandomInventoryAgentStart(InventoryAgentStart):
-    """ An inventory agentstart specification which
-    that fills
+    """ Sets the agent start inventory by randomly distributing the items throughout the inventory spaces
```

Comment:
Can you explain this further? It sounds like this just places the inventory items to random positions in inventory, which for the current MineRL version does not a affect much since the actions directly pick the first accessible item.

Reply:
Cheers! The docstring should tell that it randomly places the items in the inventory slots (the grid) a human player would see, with an additional sidenote that if you are using the inventory observations (I think all envs use these) and inventory actions, this has no effect.